youths/schema/types.py

```diff
@@ -22,14 +30,30 @@ class MembershipStatus(graphene.Enum):
     RENEWING = "renewing"
 
 
-# @extend(fields="id")
-# TODO Add separete ProfileNode with id for extending the open-city-profile profile (like in berth-reservations)
+class AdditionalContactPersonNode(DjangoObjectType):
+    class Meta:
+        model = AdditionalContactPerson
+        interfaces = (relay.Node,)
 
 
-class YouthProfileNode(DjangoObjectType):
-    membership_number = graphene.String(
-        source="membership_number", description="Youth's membership number"
-    )
+class ProfileFilter(django_filters.FilterSet):
+    class Meta:
+        model = YouthProfile
+        fields = ("membership_number",)
+
+    membership_number = django_filters.CharFilter(lookup_expr="icontains")
+
+
+@extend(fields="id")
+class ProfileNode(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ("approval_token", "language_at_home")
```

Comment:
I don't think excluding `"language_at_home"` has any affect, since the field is. defined explicitly below.
One general note: using `fields` instead of `exclude` is more reliable, since then nothing gets magically added by graphene to your GQL types, easy to overlook things that should be excluded. 😅  BUT, using `exclude` is valid, and if you don't see potential concerns, then you can keep it 👍 .

Reply:
Replacing `exclude` with `fields` is on my list of things I should refactor. Using `exclude` was/is how it was set up in open-city-profile, so here I just have not refactored it yet. But I shall at least remove `language_at_home` since it has no effect.